code/game/machinery/cryopod.dm

```diff
@@ -313,6 +313,24 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		qdel(W)//because we moved all items to preserve away
 		//and yes, this totally deletes their bodyparts one by one, I just couldn't bother
 
+	// Suspend their bank payment
+	if(mob_occupant.mind?.account_id)
+		var/number_of_bank_account_holder = 0
+		for(var/mob/living/L in GLOB.mob_list)
+			if(L?.mind?.account_id == mob_occupant.mind.account_id)
+				number_of_bank_account_holder++ // if an experimental clone exists, the account will be kept.
+				if(number_of_bank_account_holder>=2)
```

Comment:
why even check for 2, just break ALWAYS since it won't execute if there is any holders

Reply:
Because your body is yet too far deleted and is counted.

Reply:
It was to support experimental cloner, but I canceled it now because It seems bloat